actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -219,7 +219,7 @@ def stylesheet_link_tag(*sources)
             tag_options["media"] = "screen"
           end
 
-          tag(:link, tag_options)
+          ActionView.deprecator.silence { tag(:link, tag_options) }
```

Comment:
Would the following not work here?
```suggestion
          tag.link(**tag_options)
```